### DISCUSSION TITLE
Pop-up shadow color

### DISCUSSION THREAD

**Discussion by Trteich** [Clooos/Bubble-Card#1052] [Questions about config, custom styles and templates]
Is there bility to change shadow color? I can find paramrter for shadow opacity but no color.
May bbe it is possible to change it via card-mod?

**Comment by MrBearPresident** [comment]
Add this to your custom styles: 

```
.bubble-backdrop{
  background-color: #ff151540
}
```

---

<sub>If this solves your problem mark this as the solution and close the discussion. </sub>
